src/Papilio.h

```diff
@@ -0,0 +1,46 @@
+/* -*- mode: c++ -*-
```

Comment:
Similarly to the two above, I'd move it somewhere else: `src/papilio`, `src/kaleidoscope/papilio`, `src/kaleidoscope/simulator/papilio` or even `src/kaleidoscope/simulator/framework`?

Though, if it's the framework, perhaps we could just merge it with the rest of `Kaleidoscope-Simulator` into `src/kaleidoscope/simulator/`?